Backend/WebApi-PointMap/Controllers/ApplicationLaunchFromSS0Controller.cs

```diff
@@ -0,0 +1,49 @@
+﻿using DataAccessLayer.Database;

```

Comment:
This controller would be better suited to live within the existing usercontroller

Reply:
If you believe so, then easy fix